setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name            = pycyphal
-version         = file: pycyphal/VERSION
+version         = attr: pycyphal.__version__
```

Comment:
This import will fail if the library dependencies (like NumPy) are not installed. Did you mean to say `pycyphal._version.__version__`?